lib/elixir/lib/base.ex

```diff
@@ -698,7 +698,7 @@ defmodule Base do
 
   defp remove_ignored(string, nil), do: string
   defp remove_ignored(string, :whitespace) do
-    for <<c::8 <- string>>, not c in '\s\t\r\n', into: <<>>, do: <<c::8>>
+    for <<c::8 <- string>>, c not in '\s\t\r\n', into: <<>>, do: <<c::8>>
```

Comment:
The module uses `c` conventionally, so to update it here would require updating it throughout the file, which I think just mostly adds noise to the git history. WDYT?